src/runner/index.js

```diff
@@ -87,14 +99,23 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    async run ({ skipJsErrors, quarantineMode } = {}) {
+    run ({ skipJsErrors, quarantineMode } = {}) {
         this.opts.skipJsErrors   = !!skipJsErrors;
         this.opts.quarantineMode = !!quarantineMode;
 
-        var { reporterPlugin, browserSet, tests } = await this.bootstrapper.createRunnableConfiguration();
+        var getRunTaskPromise = this.bootstrapper.createRunnableConfiguration()
+            .then(({ reporterPlugin, browserSet, tests }) => {
+                this.emit('done-bootstrapping');
+
+                // NOTE: If a thenable is returned from the onResolved/onRejected handler, it will be resolved.
+                // So, in order to get a promise from a `then` block, we need to wrap it in another object.
+                return { runTaskPromise: this._runTask(reporterPlugin, browserSet, tests) };
+            });
+
+        var promise = getRunTaskPromise.then(({ runTaskPromise }) => runTaskPromise);
```

Comment:
As far as I can tell we can avoid doing this by just:
```js
return this._runTask(reporterPlugin, browserSet, tests);
```
instead of
```js
return await this._runTask(reporterPlugin, browserSet, tests);
```